roles/ceph-config/templates/ceph.conf.j2

```diff
@@ -39,11 +39,9 @@ mon initial members = {% for host in groups[mon_group_name] %}
 mon host = {% if nb_mon > 0 %}
   {% for host in groups[mon_group_name] -%}
     {% if monitor_address_block | length > 0 %}
-      {% if ip_version == 'ipv4' -%}
-        {{ hostvars[host]['ansible_all_' + ip_version + '_addresses'] | ipaddr(monitor_address_block) | first }}
-      {%- elif ip_version == 'ipv6' -%}
-        [{{ hostvars[host]['ansible_all_' + ip_version + '_addresses'] | ipaddr(monitor_address_block) | first }}]
-      {%- endif %}
+      {%- set host_addrs = hostvars[host]['ansible_all_' + ip_version + '_addresses'] -%}
```

Comment:
@Arano-kai why did you remove the ipv4/ipv6 condition?
when ipv6, the ip address must be enclosed with '[]'

Reply:
ipv6 enclosure automatically done by `ipwrap` filter at 44

Reply:
@Arano-kai in that case, it could be nice to keep something consistent by fixing it everywhere

Reply:
That will be my next push request =)

Reply:
Yes, please, let's try to not overload the CI if we can keep this in the same PR.
Thanks.

Reply:
Ok, but note that the changes will be in:

- `ceph-config/templates/ceph.conf.j2`
- `ceph-defaults/templates/restart_rgw_daemon.sh.j2`
- `ceph-mon/templates/ceph-mon.service.j2`
- `ceph-rgw/templates/ceph-radosgw.service.j2`

Reply:
Sounds good to me.